[2025-04-14T06:15:01.233+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: M3_GA_pipeline.fetch_from_postgresql manual__2025-04-14T06:14:57.795208+00:00 [queued]>
[2025-04-14T06:15:01.255+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: M3_GA_pipeline.fetch_from_postgresql manual__2025-04-14T06:14:57.795208+00:00 [queued]>
[2025-04-14T06:15:01.256+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2025-04-14T06:15:01.257+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2025-04-14T06:15:01.258+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2025-04-14T06:15:01.299+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): fetch_from_postgresql> on 2025-04-14 06:14:57.795208+00:00
[2025-04-14T06:15:01.320+0000] {standard_task_runner.py:52} INFO - Started process 461 to run task
[2025-04-14T06:15:01.325+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'M3_GA_pipeline', 'fetch_from_postgresql', 'manual__2025-04-14T06:14:57.795208+00:00', '--job-id', '39', '--raw', '--subdir', 'DAGS_FOLDER/P2M3_galuh_alifani_DAG.py', '--cfg-path', '/tmp/tmpaoijv_z3', '--error-file', '/tmp/tmpe2_zewcq']
[2025-04-14T06:15:01.328+0000] {standard_task_runner.py:80} INFO - Job 39: Subtask fetch_from_postgresql
[2025-04-14T06:15:01.573+0000] {task_command.py:371} INFO - Running <TaskInstance: M3_GA_pipeline.fetch_from_postgresql manual__2025-04-14T06:14:57.795208+00:00 [running]> on host b652548eb1f7
[2025-04-14T06:15:01.812+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Galuh
AIRFLOW_CTX_DAG_ID=M3_GA_pipeline
AIRFLOW_CTX_TASK_ID=fetch_from_postgresql
AIRFLOW_CTX_EXECUTION_DATE=2025-04-14T06:14:57.795208+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-04-14T06:14:57.795208+00:00
[2025-04-14T06:15:06.535+0000] {logging_mixin.py:115} INFO - data has been loaded
[2025-04-14T06:15:06.615+0000] {python.py:173} INFO - Done. Returned value was: None
[2025-04-14T06:15:06.660+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=M3_GA_pipeline, task_id=fetch_from_postgresql, execution_date=20250414T061457, start_date=20250414T061501, end_date=20250414T061506
[2025-04-14T06:15:06.723+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-04-14T06:15:06.808+0000] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
